includes/migration/wc/class-fs-wc-download-migration.php

```diff
@@ -92,33 +70,12 @@ function __construct(
 		 * @author Vova Feldman (@svovaf)
 		 * @since  1.0.0
 		 *
-		 * @param array $edd_price
+		 * @todo Allow user to change this somehow
 		 *
 		 * @return string
 		 */
-		private function get_billing_cycle( $edd_price ) {
-			$billing_cycle = 'lifetime';
-
-			if ( ! empty( $edd_price['recurring'] ) &&
-			     'yes' === $edd_price['recurring'] &&
-			     ! empty( $edd_price['period'] ) &&
-			     'never' !== $edd_price['period']
-			) {
-				switch ( $edd_price['period'] ) {
-					case 'year':
-						$billing_cycle = 'annual';
-						break;
-					case 'month':
-						$billing_cycle = 'monthly';
-						break;
-					default:
-						// @todo Throw an error when billing cycle is not supported.
-						$billing_cycle = 'monthly';
-						break;
-				}
-			}
-
-			return $billing_cycle;
+		private function get_billing_cycle( $id ) {
+			return 'annual';
```

Comment:
Well WooCommerce/WCAM doesn't support billing cycles... only one time payment AFAIK. I think WC Subscriptions plugin allows that...